src/components/Authentication/Login.vue

```diff
@@ -10,25 +10,35 @@
   v-container
     v-form(@submit.prevent="login", ref="login")
       v-text-field(
+          v-if="!otpFormVisible || forceOtp",
           v-model="username",
           label="Username or e-mail",
           autofocus,
           :rules="nonEmptyRules",
           prepend-icon="$vuetify.icons.user",
           type="text")
       v-text-field(
+          v-if="!otpFormVisible || forceOtp",
           v-model="password",
           type="password",
           label="Password",
           :rules="nonEmptyRules",
           prepend-icon="$vuetify.icons.lock")
-      v-layout(row)
+      v-text-field(
+          v-if="otpFormVisible || forceOtp",
+          v-model="otp",
+          type="text",
+          mask="######",
+          label="Authentication code",
```

Comment:
"Two-factor authentication code" is likely the most obvious name for the end user.

Consistently referring to this internally as a OTP is most accurate, however.